po/ja.po

```diff
@@ -8527,10 +8527,14 @@ msgid ""
 "Rust support generics, which lets you abstract algorithms or data structures "
 "(such as sorting or a binary tree) over the types used or stored."
 msgstr ""
+"Rustはジェネリクス（generics）をサポートします。これにより、使用または保存す"
+"る型に関してアルゴリズムやデータ構造（ソートアルゴリズムや、二分木など）を抽"
+"象化することができます。"
 
 #: src/generics/data-types.md:3
 msgid "You can use generics to abstract over the concrete field type:"
 msgstr ""
+"ジェネリクスを使って、フィールドを持つ具象型を抽象化することができます："
```

Comment:
Hmm, this sentence doesn't make much sense to me. Also, "concrete field type" cannot be translated to "フィールドを持つ具象型". 
If you are not so confident in your understanding or translation of a certain sentence, I would be happy if you could write about it in the comments or ask me for advice in the comments.

```suggestion
"ジェネリクスを使って、具体的なフィールドの型を抽象化することができます："
```

ジェネリクスがない`struct Point { x: u32, y: u32 }` といったstructのフィールドは`u32`のような具体的な型を持ちますが (concrete field type)、それを`T`として抽象化しているという意味です。

Reply:
Hi, @keiichiw-san, thanks for correcting me regarding that with great examples. In the original paragraph, I was a bit confused with struct Point and its nested fields' types, but as you mentioned looks like generics abstract those fields types in the context. From next MR on, I'll list unclear points in advance so that I can get feedback about it. Thank you!

Reply:
Thanks!